### ISSUE TITLE
Slow charging and battery issues 

### ISSUE THREAD

**Issue by Springfestivalxd** [PixelExperience/android-issues#2576]
## Build date

PixelExperience_phoenix-12.0-20220222-0107-OFFICIAL.zip

## Expected Behavior

The Device should charge fast because it supports 27W.

## Current Behavior

Battery taking huge time to get charge using stock charger

## Possible Solution

NOT KNOWN
One question to you wheather you left official support for phoenix? if yes then remove your name so that another guy can apply for official . Really very disappointed by PE phoenix. Last build is of FEB even twelve plus is not available if you cannot build why you took official either drop or continue please .

## Steps to Reproduce

1.Use LAST BUILD ie FEB build
2. plug charge and see
3.
4.

/device phoenix
/version twelve

**Comment by pixelexperiencebot** [comment]
Issue created! You can close at any time by commenting ```/close```

**Comment by Springfestivalxd** [comment]
@jhenrique09 
@jjpprrrr 
@ZIDAN44

**Comment by Springfestivalxd** [comment]
please let us know wheather it is dropped . same dev is making for arrow os .

**Comment by SreekanthPalakurthi** [comment]
Hey, allow me to start with charging speed. Since the very first release, the charging speed is fast enough. I made a CI build for testing out the same yesterday (16th of May 2022) and considering my ambient temperature which is close to 41°C, My device charged from 12% to 97% in 50 minutes(PFA screenshots below). Now, regarding the "Battery Issue" do note that your device is now close to two years old which makes your old beat-up battery not to perform as it did on the initial day. I'd suggest sticking with the stock kernel with no mods or tweaks to the kernel or any other magisk modules for that matter. I get around 6-7 hours of screen on time with refresh rate locked at 120Hz on a regular day with casual use with no gaming.

Now the elephant in the room, your "QUESTION".  Pushing updates for a free and open-sourced project was never on top of my to-do list and it will never be the first thing that I'd bother myself about right after I wake up in the morning, that place goes to other bigger things which hold a greater impact on my life and my future. I've already informed a core member about the same and got their permission to push an update whenever I will be free, If their decision was to drop the device for 2 months; I'd gladly do it. Anyways, I am planning to push an update by tomorrow which is the 18th of May 2022. This reply has no ill intent behind it nor I don't mean to be disrespectful towards anyone, I just wanted to clear some misconceptions. I'll be closing this issue.

![image](https://user-images.githubusercontent.com/49088464/168758756-a566f925-1ea7-46ed-958e-0193a026d82b.png)
![image](https://user-images.githubusercontent.com/49088464/168758778-93952ff0-1fa0-45f2-a360-6525a976ffbb.png)
